### DISCUSSION TITLE
Negative wind velocity values

### DISCUSSION THREAD

**Discussion by MYMahfouz** [NatLabRockies/floris#552] [Q&A]
I am doing wind farm layout optimization using FLORIS v3. I use pyoptsparse with SNOPT for the optimization algorithms.

During the optimization the algorithm moves the turbines in infeasible locations maybe too close to another turbines within the farm to find the optimum position.

I saw that the optimizer sometimes may move a turbine and the wind velocity in the flow field is negative and I had to add a negative wind value in my wind speed power and thrust coefficient input file. Otherwise the simulation fails cause it cannot interpolate for the negative wind speeds.

Why is the wind speed going to a negative value? Is this something you expect to happen?

Thanks in advance

**Comment by paulf81** [comment]
Dear @MYMahfouz , there have been some bugfixes made in recent version that may/may not address this.  Would you mind to see if pulling develop helps with this issue?  If not would you mind to post a small example which recreates the issue we can use to investigate?
